assemble/bin/accumulo-util

```diff
@@ -298,7 +298,7 @@ function create_config() {
     MAVEN_PROJ_BASEDIR="\n  <property>\n    <name>general.maven.project.basedir</name>\n    <value>${BASE_DIR}</value>\n  </property>\n"
   fi
 
-  mkdir -p "${CONF_DIR}" && cp "${TEMPLATE_CONF_DIR}"/* "${CONF_DIR}"/
+  mkdir -p "${CONF_DIR}" && cp "${TEMPLATE_CONF_DIR}"/{$ACCUMULO_SITE,$ACCUMULO_ENV} "${CONF_DIR}"/
```

Comment:
You sure this works? You can't use bash variables inside the curly braces for ranges; I'm you could use them for enumerations.